components/Story/StoryLineBanner.tsx

```diff
@@ -0,0 +1,101 @@
+import { HTMLAttributes } from "react";
+import { Container } from "@/components/Container";
+import { Heading, Text } from "@/components/Typography";
+import BannerLineart from "@/components/images/banner-lineart";
+import { cnb } from "cnbuilder";
+import { FlexBox } from "@/components/FlexBox";
+import Image from "next/image";
+
+type StoryLineBannerProps = HTMLAttributes<HTMLDivElement> & {
+  hasLineArt?: boolean;
+  heading: string;
+  superhead: string;
+  body: string;
+  byline?: string;
+  bgColor?: "stone-dark" | "red";
+  caption?: string;
+  src: string;
+  alt?: string;
+  lineVariant?: "a" | "b" | "default";
+};
+
+export const StoryLineBanner = ({
+  bgColor = "red",
+  hasLineArt = false,
+  heading,
+  superhead,
+  body,
+  byline,
+  caption,
+  src,
+  alt,
+  lineVariant,
+  ...props
+}: StoryLineBannerProps) => (
+  <Container
+    {...props}
+    as="section"
+    bgColor={bgColor}
+    width="full"
+    className="cc pt-[121px] md:rs-pt-9 lg:w-full lg:rs-px-7 mb-20 flex items-center justify-center relative 2xl:cc"
+    pb={6}
+  >
+    <div className="h-full w-full absolute top-0 left-0 z-0">
+      <Image
+        className="ed11y-ignore object-cover z-0"
+        src={src}
+        alt={alt || ""}
+        loading={"lazy"}
```

Comment:
`lazy` is the default behavior. it's not needed here. Also if the prop is a simple string, we can remove the brackets.